### DISCUSSION TITLE
SIPREC support with routr

### DISCUSSION THREAD

**Discussion by tyhu05** [fonoster/yaps#480] [Q&A] [answered]
Greetings,
May I ask if routr supports to work/integrate with a SIPREC recording server + a media server (rtpengine, etc)? I found [this thread](https://groups.google.com/g/routr-users/c/rYbxq8ZdzCQ) in routr-users group, but not sure if there is any update to support such capabilities. Thanks in advance!

**Comment by psanders** [comment, marked as answer]
Last year we added support for RTPEngine. However we didn't make it as far as testing this use case. TBH, don't know enough about this usecase to recommend a way forward.

**Reply by tyhu05** [reply]
Thank you for your clarification, Pedro.
